### PR TITLE
French, Italian and Spanish unions added

### PR DESCRIPTION
**Pull request by bcamino**
### Summary

I added the list of French, Italian and Spanish unions. For France I wasn't able to find one that specifically for science and data science, so I added the largest ones. 

### List of changes proposed in this PR (pull-request)

* *Addition of Italian, French and Spanish unions to the list*

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: <https://github.com/bcamino>

### CODE REVIEW

File: book/website/ethical-research/activism/activism-unionisation.md [hunk 1]
```diff
@@ -180,6 +180,26 @@ We recognise that the list below is very UK-centric and would welcome additions
 #### Ireland
 * **Medical Laboratory Scientists Association (MLSA)**, Ireland, is the sole trade union representing over 1600 Medical Scientists in Ireland [website](https://mlsa.ie/who-we-are)
 
+#### France
+* **Confédération Française Démocratique du Travail (CFDT)** is the largest French trade union confederation by number of members. It originated as a laic branch of the Christian trade union Confédération Française des Travailleurs Chrétiens (CFTC) [website](https://cfdt.fr/)
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-unionisation.md:184]: I think we need to find a different word for laic as I had to look this up - perhaps change this to "secular"?

File: book/website/ethical-research/activism/activism-unionisation.md [hunk 2]
```diff
@@ -180,6 +180,26 @@ We recognise that the list below is very UK-centric and would welcome additions
...
+* **Confédération Générale du Travail-Force Ouvrière (CGT-FO)** was founded by former members of the General Confederation of Labor (CGT) who accused that federation to be influenced by the French Communist Party [website](https://www.force-ouvriere.fr/?lang=fr)
+* **Confédération Française de l’Encadrement-Confédération Générale des Cadres (CFE-CGC)** is solely focused on organising unions for professional employees having higher education or in management or executive positions [website](https://cfecgc.org/)
+
+#### Italy
+* **Confederazione Generale Italiana Lavoratori (CGIL)** is the largest Italian union that counts more than 5 million members [website](http://www.cgil.it/)
+* **Confederazione Italiana Sindacato dei Lavoratori (CISL)** is based on Roman Catholic values groups linked with Christian Democracy [website](https://www.cisl.it/ )
+* **Unione Italiana Lavoratori  (UIL)** is the socialist, social democratic, (republican) and laic split from the CGIL [website](https://www.uil.it/)
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-unionisation.md:192]: Same here - laic -> secular

**Review by Arielle-Bennett — COMMENTED** [review]
Hi @bcamino, looking good! I've suggested we change 2 words as I had to look them up - let me know what you think :)

File: book/website/ethical-research/activism/activism-unionisation.md (hunk 1 above)
Comment (bcamino) [reply book/website/ethical-research/activism/activism-unionisation.md:184]: Hi @Arielle-Bennett , that sounds great. I will replace laic with secular :)

**Review by Arielle-Bennett — APPROVED** [review]

**Comment by Arielle-Bennett** [comment]
All looks good now, thanks for making the language update! ✨

**Comment by Arielle-Bennett** [comment]
@all-contributors please add @bcamino for content

**Pull request merged**